src/renderer/controllers/AuthenticationController.ts

```diff
@@ -12,34 +12,36 @@
 	export async function login(username: string, password: string): Promise<Authentication> {
 		const authFromMojang: Authentication = await uLogin({ username, password }); // official login
 		// save data to electron store
-		ApplicationStore.auth.set({ ...authFromMojang, loggedIn: true });
+		AuthStore.set({ ...authFromMojang, loggedIn: true });
 		return authFromMojang;
 	}
 	/**
 	 * Logouts user and resets store
 	 */
 	export async function logout(): Promise<void> {
-		const authData = ApplicationStore.auth.store as Authentication & { loggedIn: boolean };
-		// invalidate tokens
-		const accessToken: string = authData.accessToken;
-		const clientToken: string = authData.clientToken;
-		console.log("Invalidating access/client pair.");
-		invalidate({ accessToken, clientToken });
+		const authData = AuthStore.store;
+		if (authData.loggedIn) {
+			// invalidate tokens
+			const accessToken: string = authData.accessToken;
+			const clientToken: string = authData.clientToken;
+			console.log("Invalidating access/client pair.");
+			invalidate({ accessToken, clientToken });
 
-		// clear store
-		ApplicationStore.auth.set("loggedIn", false);
+			// clear store
+			AuthStore.set("loggedIn", false);
+		}	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5164335)